webpack/assets/javascripts/react_app/components/Bookmarks/components/SearchModal/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SearchModal from './SearchModal';
+import { useForemanModal } from '../../../ForemanModal/ForemanModalHooks';
+import { BOOKMARKS_MODAL } from '../../BookmarksConstants';
+import { translate as __ } from '../../../../common/I18n';
+import { noop } from '../../../../common/helpers';
+
+const ConnectedSearchModal = props => {
```

Comment:
is it really connected to redux?

Reply:
@laviro I would say yes, because it uses `useForemanModal` which uses `useSelector` and `useDispatch` to access Redux.

Reply:
asking also for @amirfefer @sharvit opinions,
in cases like this what do you think should be the convention? hooks are tricky ^^